packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -1587,6 +1583,29 @@ export const studioLocaleStrings = {
 
   /** Alternative text for image being shown while image is being uploaded, in previews */
   'preview.image.file-is-being-uploaded.alt-text': 'The image currently being uploaded',
+
+  /** --- Insufficient permissions message --- */
+
+  /** The title for the insufficient permissions message component */
+  'insufficient-permissions-message.title': 'Insufficient permissions',
+
+  /** Informs the users that their current roles do not have the required permissions to do a particular action */
+  'insufficient-permissions-message.not-authorized': 'Not authorized to {{action}}.',
+
+  /** Appears after the not-authorized message. Lists the current roles. */
+  'insufficient-permissions-message.roles': 'Your roles: <Roles/>',
```

Comment:
what if the user only has one Role?

Reply:
i thought it would be okay to leave it plural. my thought process: our roles system is not mutually exclusive so it signifies to the user that they can have more than one role and may need to ask for an additional one